.github/workflows/dev-checks.yml

```diff
@@ -228,28 +228,45 @@ jobs:
               const excludedDirectories = ['.github','.gitignore'];
               const modifiedFiles = response.data.files;
               const modifiedDirectories = {};
+              const newDirectories = [];
+
               response.data.files.forEach(file => {
                 const fileDirectory = file.filename.split("/")[0];
-                 if (!excludedDirectories.includes(fileDirectory)) {
-                if (!modifiedDirectories[fileDirectory]) {
-                  modifiedDirectories[fileDirectory] = false;
-                }}
+                if (!excludedDirectories.includes(fileDirectory)) {
+                  if (!modifiedDirectories[fileDirectory]) {
+                    modifiedDirectories[fileDirectory] = false;
+                    if (!excludedDirectories.includes(fileDirectory) && !newDirectories.includes(fileDirectory)) {
+                      newDirectories.push(fileDirectory);
+                    }
+                  }
+                }
               });
+
               response.data.files.forEach(function(file_entry) {
                 dir_name = file_entry.filename.split("/")[0];
                 if (file_entry.filename.split("/")[1] == 'version.txt') {
-                    console.log("Modifying version" + dir_name );
-                      if (file_entry.patch?.includes("-version=") && file_entry.patch?.includes("+version=")) {
-                           modifiedDirectories[dir_name] = true;
-                       }
+                  console.log("Modifying version" + dir_name );
+                  if (file_entry.patch?.includes("-version=") && file_entry.patch?.includes("+version=")) {
+                    modifiedDirectories[dir_name] = true;
+                  }
                 }
               });
+
               console.log(modifiedDirectories);
+
               Object.entries(modifiedDirectories).forEach(([fileDirectory, isModified]) => {
-               if (isModified == false) {
-                core.setFailed('Tessell Libraries must update the version in version.txt' + fileDirectory );
-              }
-            });
+                if (isModified == false) {
+                  if (newDirectories.includes(fileDirectory)) {
```

Comment:
This will not work. Please verify it with creating a PR with one directory having version.txt in it having the version= attribute. This should not fail.

Reply:
okay , will try